topics/single-cell/tutorials/pseudobulk-analysis/tutorial.md

```diff
@@ -0,0 +1,523 @@
+
+Several plots can be generated to assist in understanding the data and the results of the analysis, including MDS, BCV, QL, and MD plots. These visualizations provide insights into sample relationships, variability, and differential expression, and will be explained further in the tutorial. With these concepts in mind, let's now perform our DE analysis using our edgeR tool in Galaxy!
+
+> <hands-on-title> Run a DGE Analysis with edgeR </hands-on-title>
+>
+> 1. {% tool [edgeR](toolshed.g2.bx.psu.edu/repos/iuc/edger/edger/3.36.0+galaxy5) %} with the following parameters:
+>    - *"Count Files or Matrix?"*: `Single Count Matrix`
+>        - {% icon param-file %} *"Count Matrix"*: `outfile` (output of **Replace Text: Count Matrix** {% icon tool %})
+>        - *"Input factor information from file?"*: `Yes`
+>            - {% icon param-file %} *"Factor File"*: `outfile` (output of **Replace Text: Creating Factor File** {% icon tool %})
+>    - *"Use Gene Annotations?"*: `Yes`
+>        - {% icon param-file %} *"Gene Annotations"*: `output_tabular` (output of **Remove Columns: Gene Metadata** {% icon tool %})
+>    - *"Formula for linear model"*: `~ 0 + factor_A` or `~ 0 + factor_A + factor_B:factor_C`  
+>        *(Customize this formula based on your data's contrast names and factor file. Ensure the formula matches EdgeR's syntax and uses only elements from the factor file.)*
+>    - *"Input contrasts manually or through a file?"*: `file`
+>        - {% icon param-file %} *"Contrasts File"*: `outfile` (output of **Text Reformatting: Creating a Contrast File for edgeR** {% icon tool %})
+>    - In *"Filter Low Counts"*:
+>        - *"Filter lowly expressed genes?"*: `No`
+>
+>    > <comment-title> edgeR Tool Overview </comment-title>
+>    >
+>    > Use this tool to perform DGE analysis using edgeR. It is highly customizable, allowing you to input your count matrix, factor information, and gene annotations prepared during earlier steps. The contrast file should be generated from the previous step, **Text Reformatting: Creating a Contrast File for edgeR**. The model formula should be defined based on your dataset and must include factors that were already specified in the contrast file.
+>    {: .comment}
+>
+{: .hands_on}
+
+
+> <question-title>Understanding edgeR Outputs</question-title>
+>
+> 1. What are the output(s) of the edgeR tool?  
+> 2. How can we interpret our output result file?  
+>
+> <solution-title>edgeR Outputs and Interpretation</solution-title>
```

Comment:
since these boxes are nested, make sure to add a `>` in front of this line and every line below until the end of the solution box